test/mssql.jl

```diff
@@ -0,0 +1,196 @@
+@testset "mssql" begin
+    # Change this username and password to your own when testing
+    dsn = ODBC.DSN("MSSQL-test", "SA", "YourStrong!Passw0rd")
```

Comment:
Did you forget to delete this?

Reply:
Not really, but whoever runs the tests will have to specify their own so they don't really need to be included here. `SA` is the default username when using tcp, and `YourStrong!Passw0rd` the sample password microsoft gives since there are specific password rules allowed.

Reply:
Ok. I noticed the detailed comment in "runtests.jl". Probably worth moving that here.